app/views/admin/users/notes/_index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="flex flex-col h-100">
+  <h2 class="crayons-subtitle-1">Notes (<%= @notes.count %>)</h2>
+  <% if @notes.load.empty? %>
+    <div class="align-center flex flex-col justify-center my-auto py-7">
+      <h3 class="fs-xl fw-normal mb-2">🗒 No notes found.</h3>
+      <p class="color-secondary">Any notes added for @<%= @user.name %> will be found here. </p>
+    </div>
+  <% else %>
+    <div class="pt-3">
+      <% @notes.each do |note| %>
+        <article class="c-list-item flex justify-between gap-4">
+          <div>
+            <p><%= note.content %></p>
+            <p class="fs-s color-secondary">
+              <%= note.reason.presence %>
+              <% if note.author_id.present? %>
+                by <span class="fw-medium"><%= User.find(note.author_id).username %></span>
+              <% end %>
+            </p>
+          </div>
+          <time datetime="<%= note.created_at.strftime("%Y-%m-%dT%H:%M:%S%z") %>" class="color-secondary fs-s shrink-0" title="<%= note.created_at.strftime("%Y-%m-%d, %H:%M") %>">
```

Comment:
I use this in a number of places (for mails, notes, reports and flags) so maybe this should be turned into some flexible helper? This would be useful:

```
def accessible_datetime(timestamp_source, output_format, classes)
    <time datetime="<%= timestamp_source("%Y-%m-%dT%H:%M:%S%z") %>" class=classes title="<%= timestamp_source("%Y-%m-%d, %H:%M") %>">
        timestamp_source(output_format)
    </time>
end
```